Orange/modelling/base.py

```diff
@@ -68,5 +68,15 @@ def __kwargs(self, problem_type):
             self.__fits__[problem_type].__init__.__code__.co_varnames[1:])
         return {k: v for k, v in self.kwargs.items() if k in learner_kwargs}
 
+    @property
+    def supports_weights(self):
+        """The fitter supports weights if both the classification and
+        regression learners support weights."""
+        return self.get_learner(self.CLASSIFICATION).supports_weights and \
+            self.get_learner(self.REGRESSION).supports_weights
```

Comment:
Only `SklLearner` instances define `supports_weights` property, other learners do not. Will that be a problem?